src/core/server/integration_tests/saved_objects/migrations/group3/actions/actions_test_suite.ts

```diff
@@ -1120,15 +1119,17 @@ export const runActionTestSuite = ({
       `);
     });
     it('resolves left wait_for_task_completion_timeout when the task does not finish within the timeout', async () => {
-      await waitForIndexStatus({
+      const readyTaskRes = await waitForIndexStatus({
         client,
-        index: '.kibana_1',
+        index: 'existing_index_with_docs',
         status: 'yellow',
       })();
 
+      expect(Either.isRight(readyTaskRes)).toBe(true);
+
       const res = (await reindex({
         client,
-        sourceIndex: '.kibana_1',
+        sourceIndex: 'existing_index_with_docs',
```

Comment:
while we don't have any test failures from this test, it's theoretically possible that the same thing could happen here, so switching to an index that does actually exist for good measure.